python/rbac/ldap.py

```diff
@@ -43,8 +42,38 @@ def _process_extra_filter(filterstr: str) -> str:
         return filterstr
 
 
-def _get_ldap_default_settings():
+# pylint: disable=inconsistent-return-statements
```

Comment:
It's better to make return statements consistent

Reply:
fixed in https://github.com/arenadata/adcm/pull/1874